lsst-epo/investigation-static
# [C] Node version bump

Includes package version updates.

A couple packages needed to be updated to keep up with some changes related to deprecated react lifecycle methods: tested upgrading just those packages vs all and saw no conflicts so might as well bump all versions

**To Test**
1. delete node_modules folder
2. run `yarn`
3. If prompted to install new node version do so
4. If you encounter an error related to node-gyp dependencies [try this out](https://github.com/nodejs/node-gyp/issues/1927).  For me I had to install the full xcode app and then run `sudo xcode-select -s /Applications/Xcode.app/Contents/Developer` to suppress the error, but this will be wholly unique to your machine, macOS version, xcode install, etc.
5. Make sure you can run `yarn start`
5. Make sure you don't have any (new or unexpected) errors on the frontend
